controllers/util/k8sutil.go

```diff
@@ -45,6 +45,16 @@ const (
 	OwnerUIDIndexName = "ownerUID"
 
 	OdfInfoNamespacedNameClaimName = "odfinfo.odf.openshift.io"
+	OCSInitName                    = "ocsinit"
```

Comment:
Pls add a comment here that this is a ocsintialization CR names.

Reply:
done